backend/endpoints/insert.py

```
@@ -23,14 +33,104 @@ def add_to_assembly(**kwargs):
     """Adds the contents of an element to the current part studio."""
-    db = database.Database()
+    db = connect.get_db()
     api = connect.get_api(db)
     part_studio_path = connect.get_route_element_path()
-    path_to_add = connect.get_body_element_path()
+    part_studio_to_add = connect.get_body_element_path()
+    microversion_id = connect.get_body_arg("microversionId")
+    part_name = connect.get_body_arg("name")
+    configuration = connect.get_body_arg("configuration")
+
+    if configuration != None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)